docs/packages_wiki/github/june-2017.md

```diff
@@ -0,0 +1,52 @@
+---
+lang: en-us
+title: June 2017 - Monthly Planning 
+---
+
+# June 2017 - Monthly Planning 
+
+::: warning
+Please note that its possible this is outdated, as its original version was published by @'Katrina Uychaco' on Jun 14, 2017.
+:::
+
+## Looking Back 
+
+* Beta launch went very well
+* Relatively few problems
+* Performance work @smashwilson and @kuychaco did is ✨
+* GPG/credential helper work that @smashwilson shipped was solid
+* Feeling good about shipping to stable 
+
+## This Month 
+
+* Ensure room to triage and fix issues that may arise after shipping to stable 
```

Comment:
I noticed some inconsistencies with trailing white-space here. Some list-items have a trailing space. It's a bit of a delicate matter: with other file-types I'd recommend controlling this via `.editorconfig`, but since *two* trailing spaces are a feature in Markdown, EditorConfig is of no help. Not a blocker for me though.

Reply:
Maybe I could draft another PR with that feature?

Reply:
@idleberg Done and done refer to PR #16